src/app/bundles/bundle-details/bundle-details.component.html

```diff
@@ -1,3 +1,28 @@
-<p>
-  bundle-details works!
-</p>
+<app-panel [title]="'Bundle Details'" [description]="'Name bundle and provide description'"></app-panel>
+<div class="govuk-width-container">
+  <div id="bundle-details-container">
+    <form>
+      <div [className]="bundleName.touched && !bundleName.valid ? 'govuk-form-group govuk-form-group--error' : 'govuk-form-group'">
```

Comment:
I would rather move this logic out of here into a function/property that you can reference here